### DISCUSSION TITLE
Multiple Virtualizers with shared scroll parent

### DISCUSSION THREAD

**Discussion by KBHertzog** [inokawa/virtua#558] [Q&A] [answered]
I'm working on an interface that uses multiple virtualizers but they all utilize the same scroll parent. Each list has a similar set-up to this [example](https://inokawa.github.io/virtua/?path=/story/basics-virtualizer--header-and-footer). I notice that the first list displays correctly, but later lists seem to behave strangely on scroll. A number of to-dos disappear or never render at all. I tracked this down to how each Virtualizer interacts with start margin, but couldn't find a solution.

**Comment by inokawa** [comment, marked as answer]
Hi, could you explain the usecase in detail?

Probably it's not supported well by virtua. Detecting virtualizer resize with ResizeObserver and pass the size to startMargin prop may work. Or maybe we should implement the automatic start offset calculation logic of WindowVirtualizer to Virtualizer (#339 but WIP).
